httpcore5/src/main/java/org/apache/hc/core5/reactor/DefaultConnectingIOReactor.java

```diff
@@ -119,28 +115,13 @@ public IOReactorStatus getStatus() {
     }
 
     @Override
-    public List<ExceptionEvent> getExceptionLog() {
-        return auditLog.isEmpty() ? Collections.<ExceptionEvent>emptyList() : new ArrayList<>(auditLog);
+    public IOWorkers.Selector getWorkerSelector() {
```

Comment:
@clevertension Why do we need to expose `IOWorkers.Selector`? Looks completely unnecessary to me.

Reply:
change it to **protected** or **friendly**?